TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/robotMethods/RobotMethods.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
I thought we were implementing the PID into the Robot Methods subsystem (ie. changing the "move" or "update motors" methods to use a PID) and I'm not really sure what you're doing here. It feels a little overcomplicated that you're running the same calculations when you use the PID controller, too. Maybe at least write some sort of method? We can talk abt it tmr if you want to explain what you're doing though.

But it just seems unnecessary to be separating PID and RobotMethods into two things that you both need to implement in your code when trying to make your robot move, instead of just making PID something used inside of RobotMethods automatically yk

Reply:
Also I'm not quite convinced that you have it implemented in a way that's actually calling them most of these...